### ISSUE TITLE
WCAG2ICT Project: External reference links from Understanding must be added to references

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#53]
Still need to change the following links in the 1.4.3 Understanding content because neither WCAG 2.1 nor WCAG 2.2 have working reference links for these documents. Opened [WCAG issue 2756](https://github.com/w3c/wcag/issues/2756) to document these missing links from the WCAG understanding documents.

- <a href="http://www.w3.org/TR/2013/NOTE-UNDERSTANDING-WCAG20-20130905/appendixD.html#ISO-9241-3">[ISO-9241-3]</a> 
- <a href="http://www.w3.org/TR/2013/NOTE-UNDERSTANDING-WCAG20-20130905/appendixD.html#ANSI-HFES-100-1988">[ANSI-HFES-100-1988]</a>
- <a href="http://www.w3.org/TR/2013/NOTE-UNDERSTANDING-WCAG20-20130905/appendixD.html#ARDITI-FAYE">[ARDITI-FAYE]</a>
- <a href="http://www.w3.org/TR/2013/NOTE-UNDERSTANDING-WCAG20-20130905/appendixD.html#ARDITI-KNOBLAUCH-1994">[ARDITI-KNOBLAUCH]</a>
- <a href="http://www.w3.org/TR/2013/NOTE-UNDERSTANDING-WCAG20-20130905/appendixD.html#ARDITI-KNOBLAUCH-1996">[ARDITI-KNOBLAUCH-1996]</a>
- <a href="http://www.w3.org/TR/2013/NOTE-UNDERSTANDING-WCAG20-20130905/appendixD.html#ARDITI">[ARDITI]</a>
- <a href="http://www.w3.org/TR/2013/NOTE-UNDERSTANDING-WCAG20-20130905/appendixD.html#GITTINGS-FOZARD">[GITTINGS-FOZARD]</a>
- <a href="http://www.w3.org/TR/2013/NOTE-UNDERSTANDING-WCAG20-20130905/appendixD.html#IEC-4WD">[IEC-4WD]</a> 
- <a href="http://www.w3.org/TR/2013/NOTE-UNDERSTANDING-WCAG20-20130905/appendixD.html#sRGB">[sRGB]</a>

_Originally posted by @maryjom in https://github.com/w3c/wcag2ict/issues/18#issuecomment-1295511697_
